src/Tizen.NUI/src/internal/WebView/WebContextMenu.cs

```diff
@@ -63,19 +63,6 @@ public WebContextMenuItemList ItemList
             }
         }
 
-        /// <summary>
-        /// Gets position of the context menu.
-        /// </summary>
```

Comment:
The VD requirements list include "ewk_context_menu_pos_x_get", "ewk_context_menu_pos_y_get".
I thought they are matched with WebContextMenu.Position.
After removing WebContext.Position, which API is patched with the VD requirements?

Reply:
Mr Youngha Jung said in the mail:
"ewk_context_menu_pos_x_get and ewk_context_menu_pos_y_get return the position of the context menu created using elm popup.
In your case, I think you can return the position of the nui popup."

Both APIs are used for elm popup, not required for NUI.

Reply:
Are we using NUI popup to post context menu (or should we implement)? I thought the context menu appears inside of webview so it is rendered by web engine.

Reply:
Context menu popup could be implemented by NUI apps, just like JavaScript alert/prompt/confirm popups.
When apps users long press or do something else, ContextMenuShown event would be triggered.
When ContextMenuShown event is received, NUI apps would get menu information from WebContextMenu,
for example, how many menu items would be shown, then a popup would be created and shown.
If elm popup is shown, its style is different from that of NUI. And NUI apps might have their own UI styles, 
So I followed the comments of Mr Youngha Jung, and removed WebContext.Position in dali and NUI.

Reply:
Could you add example of creating context menu using NUI controls?

Reply:
I added a MenuView into test/Tizen.NUI.WebViewTest/SimpleWebViewApp.cs .
MenuView is very simple. It would be shown when touched, hidden when touched again.